packages/mc-html-template/src/load-html-docs.js

```diff
@@ -0,0 +1,21 @@
+// @preval
+/**
+ * https://github.com/kentcdodds/babel-plugin-preval#preval-file-comment--preval
+ * NOTE: This file is pre-evaluated during build time, using `babel-plugin-preval`.
+ * This is ok as the loaded files are static anyway and it prevents possible
+ * loading issues when files are required through Webpack own context.
+ */
```

Comment:
This is important and it's something we could have done some time ago.

Essentially instead of shipping the actual HTML, CSS, JS snippets, we bundle them inline on compile time.

This is how this file looks:

<img width="730" alt="image" src="https://user-images.githubusercontent.com/1110551/165335070-e12eb378-3336-417e-b808-cb111bc28c61.png">